### PR TITLE
[Improve][Doc] Add new connector to connector v2 release state doc

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

**Review by TaoZex — DISMISSED** [review]

File: docs/en/Connector-v2-release-state.md [hunk 1]
```diff
@@ -55,3 +55,8 @@ SeaTunnel uses a grading system for connectors to help you understand what to ex
 | [Sentry](connector-v2/sink/Sentry.md)                       | Sink   | Alpha  | 2.2.0-beta      |
 | [Socket](connector-v2/sink/Socket.md)                       | Sink   | Alpha  | 2.2.0-beta      |
 | [Socket](connector-v2/source/Socket.md)                     | Source | Alpha  | 2.2.0-beta      |
+| [Kafka](connector-v2/source/kafka.md)                       | Source | Alpha  | 2.3.0-beta      |
+| [Kafka](connector-v2/sink/Kafka.md)                         | Sink   | Alpha  | 2.3.0-beta      |
+| [Pulsar](connector-v2/source/pulsar.md)                     | Source | Alpha  | 2.3.0-beta      |
```
Comment (ashulin) [inline docs/en/Connector-v2-release-state.md]: Repeat with line 52

**Review by ic4y — DISMISSED** [review]
+1

**Review by TaoZex — DISMISSED** [review]
LGTM

**Review by ic4y — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Review by TaoZex — APPROVED** [review]

**Review by ashulin — APPROVED** [review]
+1

**Pull request merged**